app/javascript/src/components/category.vue

```diff
@@ -41,17 +41,13 @@
       <p
         class="range"
       >
-        <span class="inline-block">
-          <span class="inline-block w-6 h-6 border border-gray-500 border-solid rounded-full range__circle">$</span>
-          : 5000 - 10000
-        </span>
-        <span class="inline-block">
-          <span class="inline-block w-6 h-6 ml-5 border border-gray-500 border-solid rounded-full range__circle">$$</span>
-          : 10000 - 15000
-        </span>
-        <span class="inline-block">
-          <span class="inline-block w-6 h-6 ml-5 border border-gray-500 border-solid rounded-full range__circle">$$$</span>
-          : 15000 - 20000
+        <span
+          v-for="i in (priceIntervals.length - 1)"
+          :key="i"
+          class="inline-block ml-5 first:ml-0"
+        >
+          <span class="inline-block w-6 h-6 border border-gray-500 border-solid rounded-full range__circle"> {{ '$'.repeat(i) }}</span>
+          {{ `: ${priceIntervals[i-1]} - ${priceIntervals[i]}` }}
```

Comment:
me queda una duda acá, aca para el primer intervalo quedaria algo como 

`- 10000` o no? porque estas partiendo desde `i = 0` y `priceIntervals[i-1] === undefined`

Reply:
el v-for con rango parte desde 1, de hecho tuve que ir a la [documentación](https://es.vuejs.org/v2/guide/list.html#v-for-con-un-Rango) para comprobarlo jaja

Reply:
😮  nice!